content/apps/databases.md

```diff
@@ -63,35 +63,49 @@ To access a service database, use the [cf-ssh]({{< relref "getting-started/one-o
 
 You should now have an open `psql` terminal connected to the service database.
 
-### Accessing files created on the server using cf files
+## Export
 
-When accessing a database using `cf-ssh`, you may produce a database dump file
-and will very likely need to pull it down from the server. You can easily print
-out any file on your server to your terminal via the `cf files` command. You
-have to make sure you haven't disconnected from your ssh session that you
-created with `cf-ssh`.
+### Create backup
+
+First, spin up a host to connect to the database:
+
+```sh
+$ cf-ssh <app name>
+```
+
+When it finishes, you'll be in a tmux connection and you can create the export file:
+
+```sh
+$ pg_dump $DATABASE_URL > /tmp/backup.sql
+```
+
+Leave the ssh connection open.
+
+
+### Download
+
+On your local host:
 
 ```sh
-cf files <APP_NAME> [PATH] [-i INSTANCE]
+$ cf files <app name> /tmp/backup.sql > backup.sql
```

Comment:
@dogweather should we add the `-ssh` here? I think that is missing now that the following paragraph is killed